examples/text.rs

```diff
@@ -0,0 +1,24 @@
+use cherry::widget::{
+    text::{Options, Text},
+    Widget,
+};
+use embedded_graphics::{
+    mono_font::{ascii::FONT_10X20, MonoTextStyle},
+    pixelcolor::Rgb888,
+    prelude::{Point, RgbColor, Size},
+};
+use embedded_graphics_simulator::{OutputSettingsBuilder, SimulatorDisplay, Window};
+use std::convert::Infallible;
```

Comment:
What is `Infallible`?

Reply:
I haven't read about it, so speculating a bit, but I think the display trait that `embedded_graphics` uses requires drawing calls to return a result, since they can fail (for hardware or whatever reasons), but since the simulator will never fail, it uses an unconstructable type (`Infallible`) as it's error. (So I don't think you could even match against it?)